backend/serializers.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import User
+
+from rest_framework import serializers
+from rest_auth.registration.serializers import RegisterSerializer
+
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+
+class AuthRegisterSerializer(RegisterSerializer):
+
+    def get_cleaned_data(self):
+        return {
+            'username': self.validated_data.get('username', ''),
+            'password1': self.validated_data.get('password1', ''),
+            'email': self.validated_data.get('email', ''),
+            'is_active': False
+        }
+
+    def save(self, request):
+        # Get the cleaned JSON data 
+        adapter = get_adapter()
+        user = adapter.new_user(request)
+        user.is_active = False
+        self.cleaned_data = self.get_cleaned_data()
```

Comment:
where is this `cleaned_data` variable used?

Reply:
it is used in the apiview from the library